dashboard/app/models/sections/section.rb

```diff
@@ -283,7 +283,7 @@ def summarize(include_students: true)
       currentUnitTitle: title_of_current_unit,
       linkToCurrentUnit: link_to_current_unit,
       numberOfStudents: num_students,
-      linkToStudents: "#{base_url}#{id}/manage",
+      linkToStudents: "#{base_url}#{id}/manage_students",
```

Comment:
Do you know if this link is used / visible anywhere?  I'm wondering if a teacher might have recently saved the old link, in which case we might want to ask if it's ok for it to "suddenly" stop working.

Reply:
We're not linking directly here from anywhere I can find on the site. It's possible that someone who's been teaching with Code.org for a long time (2+ years) has this link bookmarked and always goes through the redirect to get to the current teacher dashboard.